### ISSUE TITLE
I want to download video and audio as mp4 and wav but the downloader gives me both files as webm instead!

### ISSUE THREAD

**Issue by Philhubk98** [stefnotch/downline#36]
Also the downloader wouldn't download to the exact location where I want it and always on the Downline folder. Please fix this!

**Comment by stefnotch** [comment]
Hi

Yeah, the current version has some frustrating bugs. The problem is that I won't fix them anytime soon, since I'm not interested in working on this particular project all by myself.

My recommendations to the reader would be
- Volunteer to contribute, or find someone who might be interested in contributing. I'll gladly guide contributors to the best of my abilities.
- Find an alternative downloader that is better in every aspect. (Easy to install, small file size, ...) If you do find one, please tell me about it.

**Comment by Philhubk98** [comment]
Well thats very disappointing to hear. Are you saying you are no longer interested with this tool you just made yourself?! 
You should realized how very useful and handy it is! I just uninstall Videoder because the downloading is too slow and sometimes have errors, stopping my downloads but thanks to this wonderful amazing tool of yours its fast and doesn't have errors except the required file to download Youtube videos was for whatever reason wasn't set up properly and I end up fixing it myself by clicking a few things to put it together myself. This would be easier if you know how to fix it around and patch it for us! 
Just giving up your project like this would be a lazy choice when you could continue it to bring us a useful replacement with crummy old Videoder!  
All other alternatives are useless garbage, yours was no doubt the best but Im afraid not many will be able to support due to how less known this tool really is honestly.  
Please consider improving it and fixing it yourself for us! 
Best regards and thanks so much for the tool!    On Tuesday, December 5, 2023 at 05:03:29 PM GMT+8, stefnotch ***@***.***> wrote:  
 
 


Hi

Yeah, the current version has some frustrating bugs. The problem is that I won't fix them anytime soon, since I'm not interested in working on this particular project all by myself.

My recommendations to the reader would be
   
   - Volunteer to contribute, or find someone who might be interested in contributing. I'll gladly guide contributors to the best of my abilities.
   - Find an alternative downloader that is better in every aspect. (Easy to install, small file size, ...) If you do find one, please tell me about it.

**Comment by stefnotch** [comment]
Aw, thanks for being so interested in this project. As a developer, it's a super nice feeling knowing that there are other users.

Anyways, it's not necessarily that I'm not interested in working on it. It's more of me being unable to quite justify putting in the amount of work that would be required to make this tool truly good. Kinda my own fault for having high standards, I suppose.


Finally, in the spirit of actually attempting to solve your problem: Have you had a chance to look at https://www.reddit.com/r/youtubedl/wiki/info-guis/ ?
Basically, that's a list of programs that use [the same underlying downloader](https://github.com/yt-dlp/yt-dlp). The list is hopefully still up to date, since I personally helped with updating it about a year ago.
